samples/MASMessagingSample/build.gradle

```diff
@@ -33,10 +33,13 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    testCompile 'junit:junit:4.12'
     compile "com.android.support:recyclerview-v7:$rootProject.ext.recyclerViewVersion"
     compile "com.android.support:design:$rootProject.ext.designVersion"
+    compile 'pl.droidsonroids.gif:android-gif-drawable:1.2.7'
```

Comment:
I'll be removing the pigeon screen and adding a simple screen to enter the app which will have a normal button. 
As for the logout button, I picked it from internet, the site didn't asked for any permission. Anyhow I'll ask one of my colleague here who is expert in making icons to make me a new one.